### PR TITLE
update rollup

### PR DESCRIPTION
**Pull request by kellyselden**
Unblocks #140

Not sure why the test change was necessary, but the new assertion tests the same thing.

### CODE REVIEW

**Comment by lukastaegert** [comment]
I have opened an issue with rollup to investigate this further as it seems you have stumbled across a bug here: rollup/rollup#1992

**Comment by guybedford** [comment]
I just cloned this and linked in the current build of Rollup and it is definitely working for me with that test.

**Comment by guybedford** [comment]
Ah, spoke too soon... looking into it :)

File: test/test.js [hunk 1]
```diff
@@ -369,7 +369,7 @@ describe( 'rollup-plugin-node-resolve', function () {
 				})
 			]
 		}).then( bundle => {
-			assert.deepEqual( bundle.imports.sort(), [] );
+			assert.equal( bundle.modules.length, 2 );
```
Comment (lukastaegert) [inline test/test.js]: The latest release of rollup should fix these test regressions.

**Comment by kellyselden** [comment]
I reverted the test changes.

**Pull request merged**